spec/fixtures/api/leak-exit-browserview.js

```diff
@@ -0,0 +1,6 @@
+const { BrowserView, app } = require('electron')
+app.on('ready', function () {
+  new BrowserView({})  // eslint-disable-line
+
+  process.nextTick(() => app.quit())
```

Comment:
Any reason this has to be on the next tick ?

Reply:
I'm trying to emulate the original crash case, having `process.nextTick` or not does not really affect the test.